model/cli/spans.yaml

```
@@ -0,0 +1,21 @@
+groups:
+  - id: span.cli
```

I need help with this for pleasing policies-check

---

it  should be `span.cli.internal`

```suggestion
  - id: span.cli.internal
```